themes/ee/asset/javascript/src/cp/form_validation.js

```diff
@@ -236,7 +236,6 @@ EE.cp.formValidation = {
 	 * @param	{jQuery object}	form	jQuery object of form
 	 */
 	_errorsExist: function(form) {
-
```

Comment:
@Yulyaswan when possible, try to avoid pushing changes like adding/removing empty line or space (if that does not improve code readability). This will help to mantain cleaner Git history